### ISSUE TITLE
Some games are not being added/detected by the scanner

### ISSUE THREAD

**Issue by xxxarmitagexxx** [moraroy/NonSteamLaunchers-On-Steam-Deck#507]
Hi,

I have installed NonSteamLaunchers through Decky and installed a few launcher EA/Ubisoft Connect/Gog.
So far installed a couple of games through the EA launchers: 
- Mass Effect legendary: error on the EA launcher, won't strat
- Mirror's edge: installed fine but was not added as a non steam game, it launchs and play fine when launched from the EA launcher.

I have tried to manually scan for games as well, at it is never seen.
Any work-around to add a game installed through a NonSteamLaunchers and that is not automatically detected?

**Comment by moraroy** [comment]
@xxxarmitagexxx is this still an issue for you even with the latest version of the plugin, using NSLDesktop.plugin?

**Comment by lucasmouragomes** [comment]
This has happened to me as well.

Installed GRIP - Combat Racing through GOG, but it hasn't beeb added. 

GRIP folder is in GOG games with Disco Elysium and Children of Morta. The other two show up, but GRIP does not.

I tries both auto scan and manial scan

**Comment by moraroy** [comment]
@lucasmouragomes do you have the latest version of the plugin?

**Comment by lucasmouragomes** [comment]
Yes, I had just installed it. Manual scanning didn't solve either.

**Comment by lucasmouragomes** [comment]
[Software\\Wow6432Node\\GOG.com\\Games\\1342466874] 1734397973
#time=1db5020cc1ddcbe
"BUILDID"="54811501048661455"
"dependsOn"=""
"DLC"=""
"exe"="C:\\Program Files (x86)\\GOG Galaxy\\Games\\GRIP\\Grip\\Binaries\\Win64\\Grip-Win64-Shipping.exe"
"exeFile"="Grip-Win64-Shipping.exe"
"gameID"="1342466874"
"gameName"="GRIP"
"INSTALLDATE"="2024-12-16 22:12:53"
"installer_language"="english"
"lang_code"="en-US"
"language"="english"
"launchCommand"="C:\\Program Files (x86)\\GOG Galaxy\\Games\\GRIP\\Grip\\Binaries\\Win64\\Grip-Win64-Shipping.exe -culture=en-EN"
"launchParam"="-culture=en-EN"
"path"="C:\\Program Files (x86)\\GOG Galaxy\\Games\\GRIP"
"productID"="1342466874"
"savegamefolder"="C:\\users\\steamuser\\AppData\\Local\\GRIP"
"startMenu"="GRIP"
"startMenuLink"="C:\\ProgramData\\Microsoft\\Windows\\Start Menu\\Programs\\GRIP [GOG.com]\\GRIP"
"supportLink"="http://www.gog.com/support/grip"
"uninstallCommand"="C:\\Program Files (x86)\\GOG Galaxy\\Games\\GRIP\\unins000.exe"
"ver"="1.5.3 v2"
"workingDir"="C:\\Program Files (x86)\\GOG Galaxy\\Games\\GRIP\\Grip\\Binaries\\Win64"

this is the entry at system.reg

**Comment by moraroy** [comment]
@lucasmouragomes it looks normal to me... I'm thinking maybe the dashes - in the exe name and it's not picking it up,  not sure yet! Give me some time I'll try to see what's going on. In the mean time try uninstalling the game and reinstalling if you can....also.... inside the gog galaxy launcher itself, very/repair the game and see if you can even "add it manually" that should be on the top left of the gog galaxy launcher. Gog allows you to search folders. Let me know if if amy of that works, it may simply be a typo because of the dashes. Print strings/logger.info would need to be added to certain parts of the scanner in order to figure out why

**Comment by danzeff** [comment]
I'm having a similar problem. I installed two games via the EA app: Spore and Simcity 4. The former appeared after a manual scan, the latter did not. I've already tried to repair the install but still no luck.

**Comment by moraroy** [comment]
@danzeff did you install them on am SD card or default location?

**Comment by danzeff** [comment]
@moraroy Default location

**Comment by moraroy** [comment]
@danzeff also just a heads up, I really need to re-do the battle.net scanner I feel like it's a hit or miss. It's kind of hard to parse, especially when new games come out.

**Comment by moraroy** [comment]
@danzeff and possibly the ea app... are you on Desktop or plugin?

**Comment by danzeff** [comment]
@moraroy I use the desktop version
